test/unit/objects/test_model.py

```diff
@@ -209,7 +213,7 @@ def test_jtracedist(self):
 
     @needs_cvxpy
     def test_diamonddist(self):
-        if SKIP_DIAMONDIST_ON_WIN and sys.platform.startswith('win'): return
+        if  SKIP_DIAMONDIST_ON_WIN and sys.platform.startswith('win'): return
```

Comment:
Can you do a merge of develop into your branch. This line was removed in PR #588 which was merged in a short while ago, so as is this would unintentionally add it back in.